html/mycakeapp/webroot/css/auction.css

```
@@ -24,4 +24,138 @@ table tr th.small {
 }
 nav.sidebar {
 	background:#f6f6ff;
```

（修正不要）
自分で書いたところと、もともとの部分があるので致し方ない部分もありますが…
プロパティと値の間は、1つ空白を開けたほうが見やすいですね。

```suggestion
	background: #f6f6ff;
```

※ 1つあけなさい、という指摘というよりは、少なくとも1ファイル（できればプロジェクト内）では統一しましょう、という意図が強い指摘です。

---

ご丁寧にご指摘ありがとうございます！
以後プロジェクトごとに統一するように注意します！